install/kubernetes/helm/subcharts/gateways/templates/deployment.yaml

```diff
@@ -54,6 +54,19 @@ spec:
             privileged: true
 {{- end }}
       containers:
+{{- if eq $key "istio-ingressgateway" }}
```

Comment:
Why ? Any gateway should be able to do this.

Reply:
Shall we enable this feature at ingress gateway for now? As the design doc focuses on ingress gateway. If it works fine then we can enable it at other gateways.

Reply:
'ingressgateway' is just a name. User can have multiple gateways - each gateway is associated with an IP and namespace.  All those gateways will be used for ingress - but have different names and IP addresses ( and possibly different namespaces ).